content/blog/2016/2016-12-13-jenkins-puppet-enterprise-plugin.adoc

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Continuous delivery with Jenkins and Puppet Enterprise"
+tags:
+- jenkins
+- continuousdelivery
+- puppet
+author: ccaum
+---
+
+During PuppetConf 2016, myself and Brian Dawson from CloudBees announced the link:https://wiki.jenkins-ci.org/display/JENKINS/Puppet+Enterprise+Pipeline+Plugin[Puppet Enterprise plugin for Jenkins Pipeline.] Let's take a look at how the plugin makes it trivial to use Puppet to perform some or all of the deployment tasks in continuous delivery pipelines.
```

Comment:
Fair enough. I though it wouldn't matter once it was rendered to HTML, but you're right. All fixed.